test/fixtures/e2eapi.go

```diff
@@ -51,12 +51,10 @@ func InvokeE2EAPI(format string, args ...interface{}) string {
 
 func InvokeE2EAPIPOST(format string, body string, args ...interface{}) string {
 	url := "http://127.0.0.1:8378" + fmt.Sprintf(format, args...)
-	log.Printf("Post %s\n", url)
```

Comment:
Removing this log to make logs cleaner in e2e tests. Our reduce tests are posting a lot of requests, which adds noise in the logs and it's not helpful to debug.